doc/meetings/2019-05-09.md

```diff
@@ -0,0 +1,56 @@
+# Node.js Foundation Release WorkGroup Meeting 2019-05-09
+
+## Links
+
+* **GitHub Issue**: https://github.com/nodejs/Release/issues/444
+* **Minutes Google Doc**: https://docs.google.com/document/d/1wilsRUSzbsqk-D1pG2P3LEplg_UosPPtZwrk-8yPbyY
+
+## Present
+
+* Beth
+* Richard
+* Ruben
+* Sam
+* Shelley
+
+
+## Agenda
+
+## Announcements
+ 
+*Extracted from **Release-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/Release
+
+* Adding a "security" property to dist/index.json [#437](https://github.com/nodejs/Release/issues/437)
+    - Richard is working on this. 
+    - **Action:** Release team to take a look at https://github.com/nodejs/node/pull/27612 and https://github.com/nodejs/nodejs-dist-indexer/pull/8 and provide feedback. 
+* Defining a word for all current releases [#359](https://github.com/nodejs/Release/issues/359)
+    - Discussed alternative words. Suggestions: LTS Maintenance, Current Maintenance. 
+    - **Action:** Will reignite discussion on the issue.
```

Comment:
Ruben suggested next step would be PRing actual text, to make comment and iteration on terminology easier/more focussed.